camayoc/tests/qcs/cli/utils.py

```diff
@@ -155,6 +156,48 @@ def source_show(options, output, exitstatus=0):
     assert qpc_source_show.exitstatus == exitstatus
 
 
+def scan_add(options):
```

Comment:
I like the idea of having another level of helper methods where some assertions are done. But I would stick with having helper methods named after qpc commands doing all the same and create another set for helpers that performs assertions.